reporter/src/funTest/assets/spdx-document-reporter-expected-output.spdx.json

```diff
@@ -0,0 +1,14 @@
+{
+  "SPDXID" : "SPDXRef-<REPLACE_UUID>",
+  "spdxVersion" : "SPDX-2.2.1",
+  "creationInfo" : {
+    "comment" : "some creation info comment",
+    "created" : "1970-01-01T00:00:00Z",
+    "creators" : [ "Tool: OSS Review Toolkit - 0.1" ],
+    "licenseListVersion" : "3.9"
+  },
+  "name" : "some document name",
+  "dataLicense" : "CC0-1.0",
+  "comment" : "some creation info comment",
+  "documentNamespace" : "spdx://<REPLACE_UUID>"
+}
```

Comment:
The file should end with a newline.

Reply:
Done.

Reply:
Hmm, for me GitHub shows a missing newline.

Reply:
True - I've undone the change as the tests do fail with a new-line at the end, and for consistency with other `*expected-result*.json` files like `evaluated-model-reporter-test-expected-output.json`.

Reply:
> I've undone the change as the tests do fail with a new-line at the end

Ah, I recall we already had this [issue with Jackson's JSON pretty printer](https://stackoverflow.com/q/33040791/1127485).